pinot-common/src/main/java/com/linkedin/pinot/common/data/StarTreeIndexSpec.java

```diff
@@ -96,4 +101,8 @@ public void setExcludeSkipMaterializationDimensionsForStarTreeIndex(
   public String toString() {
     return ToStringBuilder.reflectionToString(this, ToStringStyle.SHORT_PREFIX_STYLE);
   }
+
+  public String toJsonString() throws Exception {
```

Comment:
I know this is already committed, but a couple of suggestions:
1. Use static final ObjectMapper so no need to new each time.
2. I use objectMapper().writerWithDefaultPrettyPrinter() to make the json output pretty printed.

Reply:
thanks, https://github.com/linkedin/pinot/pull/2134